### PR TITLE
Update scipy_proc.json for 2024

### PR DESCRIPTION
**Pull request by ameyxd**
Update 2024 OC Members

If you are creating this PR in order to submit a draft of your paper,
see http://procbuild.scipy.org/ for logs generated by the build
process.

See the [project readme](https://github.com/scipy-conference/scipy_proceedings#build-your-paper)
for more information.

*Editor*:  

*Reviewers*:

### CODE REVIEW

File: scipy_proc.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
       "prefix": "10.25080",
       "registrant": "Crossref",
       "resource_url": "https://conference.scipy.org/proceedings",
-      "depositor_name": "Dillon Niederhut",
-      "depositor_email": "dillon.niederhut@gmail.com"
+      "depositor_name": "Amey Ambade",
+      "depositor_email": "ameyambade@gmail.com"
```
Comment (rowanc1) [inline scipy_proc.json]: Can this go to the proceedings group email address?

Comment (ameyxd) [reply scipy_proc.json]: Can you share that address or recommend that change in the file? @rowanc1

Comment (rowanc1) [inline scipy_proc.json]:
```suggestion
      "depositor_name": "SciPy Proceedings",
      "depositor_email": "proceedings@scipy.org"
```

File: scipy_proc.json [hunk 2]
```diff
@@ -19,30 +19,30 @@
...
       "rgoswami@quansight.com",
-      "dillon.niederhut@gmail.com"
+      "ameyambade@gmail.com"
     ],
     "dates": "July 10 - July 16",
     "editor": [
       "Meghann Agarwal",
       "Chris Calloway",
       "Rohit Goswami",
-      "Dillon Niederhut"
```
Comment (mepa) [inline scipy_proc.json:39]: Below is the current list of 2024 proceedings committee co-chairs. Should we go ahead and update this now? If so, please feel free to re-order another way - the following is in alphabetical order by last name.
```
"Meghann Agarwal",
"Amey Ambade",
"Chris Calloway",
"Rowan Cockett",
"Sanhita Joshi",
"Charles Lindsey",
"Hongsup Shin"
```

File: scipy_proc.json [hunk 3]
```diff
@@ -19,30 +19,30 @@
...
       "acronym": "SciPy",
-      "ordinal": "22nd"
+      "ordinal": "23nd"
     },
     "citation_key": "proc-scipy-2023",
     "editor_email": [
       "agarwal.meghann@gmail.com",
       "cbc@unc.edu",
       "rgoswami@quansight.com",
-      "dillon.niederhut@gmail.com"
```
Comment (mepa) [inline scipy_proc.json:32]: Below is the corresponding list of e-mail addresses.
```
"agarwal.meghann@gmail.com",
"ameyambade@gmail.com",
"cbc@unc.edu",
"rowan@curvenote.com",
"sanhita.joshi@gmail.com",
"lindseycster@gmail.com",
"hongsup.shin@pm.me"
```

**Comment by ameyxd** [comment]
@mepa you may review the updated commit.

File: scipy_proc.json [hunk 4]
```diff
@@ -219,12 +249,65 @@
...
+    },
+    {
+      "name": "Data Visualization and Image Processing",
+      "members": [
+        {
+          "org": "NVIDIA",
+          "name": "John Kirkham"
+        }
+      ]
+    },
```
Comment (mepa) [inline scipy_proc.json]: I think we should nest these as a list of values associated with a new key (e.g., `"program_track_chairs"`) that front-end code could easily parse and display together (e.g., see section "Track & Mini Symposia Chairs" [here](https://www.scipy2022.scipy.org/organizers)).

This is the current json structure:
Screenshot 2024-04-02 at 19 01 22

Proposed future json structure:
Screenshot 2024-04-02 at 19 18 28

Comment (ameyxd) [reply scipy_proc.json]: done for now

**Comment by mepa** [comment]
Thanks, @ameyxd! It looks good to me.

**Review by mepa — APPROVED** [review]

**Comment by ameyxd** [comment]
Are we good to merge this @mepa

**Pull request merged**